src/System.Data.SqlClient/src/project.json

```diff
@@ -13,7 +13,7 @@
     "System.IO.FileSystem": "4.0.0",
     "System.Net.Primitives": "4.0.10",
     "System.Net.Security": "4.0.0-beta-*",
-    "System.Net.Sockets": "4.0.10-beta-*",
+    "System.Net.Sockets": "4.1.0-beta-*",
```

Comment:
When are these changes targeted to be released? 
Will there be sufficient time for us to react and make changes in the RC1 timeframe?